src/test/java/uk/gov/hmcts/reform/bulkscanprocessor/services/reports/ReportsServiceTest.java

```diff
@@ -163,11 +163,11 @@ public void should_filter_zipfiles_by_jurisdiction_when_requested_for_zipfiles_s
             ));
 
         // when
-        List<ZipFileSummaryResponse> result = service.getZipFilesSummary(now(), "J2");
+        List<ZipFileSummaryResponse> result = service.getZipFilesSummary(now(), "j2");
```

Comment:
did you intend to change it to `c2`?

Reply:
no.. but this looks better. I'll change it to `c2`